src/E1_coldflow/E1_coldflow.cpp

```diff
@@ -12,7 +12,7 @@
 #include <tempController.h>
 #include <batteryMonitor.h>
 
-#define SERIAL_INPUT 0 // 0 is flight config, 1 is for debug
+#define SERIAL_INPUT 1 // 0 is flight config, 1 is for debug
```

Comment:
we should always make sure the committed version has SERIAL_INPUT set to 0, so that we can pull and upload to the flight computer without changing anything.